micromeow/fosterfinder
# Feat: Add Pagination

## Description

This PR adds a new `app/animals` page and a pagination component to complement it

## Changes
- Add `Pagination` with tests and stories
- Add `PaginationWithHooks` to read `searchParams`
- Add `app/animals` page to display a list of animals available
- Clean up font import by importing via `next/font/google` rather than css directly

## Testing
1. `npm i`
2. `npm run dev`
3. `npm run db:reset`
4. Go to [/app/animals](http://localhost:3000/app/animals)
5. Pagination should be intuitive and easy to use
6. `npm run storybook`
7. Click on `Pagination/Docs`
8. Properties should be easy to understand

## Screenshots
<img width="539" alt="image" src="https://github.com/micromeow/fosterfinder/assets/13270108/9fa5971c-4c9a-4f20-9bb3-0548427caa30">
